test/support/flutter_test_driver.dart

```diff
@@ -431,6 +431,18 @@ class FlutterRunTestDriver extends FlutterTestDriver {
     return 0;
   }
 
+  Future<dynamic> callServiceExtension(
```

Comment:
why can't we just use the vmService for this? I've lost the thread a little bit on this part of the code.

Reply:
You're right, we can. Removed method from flutter_test_driver and updated tests to use vmService.